src/main/java/seedu/address/logic/commands/lesson/AddLessonCommand.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.logic.commands.lesson;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CODE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TIME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TYPE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_WEIGHTAGE;
+
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.lesson.Lesson;
+
+public class AddLessonCommand extends Command {
+
+    public static final String COMMAND_WORD = "add";
+
+    public static final String MESSAGE_USAGE = Lesson.TYPE + " " + COMMAND_WORD
+            + ": Adds a lesson to TrackIt@NUS. "
+            + "Parameters: "
+            + PREFIX_CODE + "MODULE_CODE "
+            + PREFIX_TYPE + "TYPE "
+            + PREFIX_TIME + "DATE "
+            + PREFIX_ADDRESS + "LOCATION "
+            + PREFIX_WEIGHTAGE + "WEIGHTAGE\n";
```

Comment:
Maybe can add an example, like in AddContactCommand

Reply:
Noted with thanks!